src/test/java/org/openstreetmap/atlas/checks/validation/intersections/LineCrossingWaterBodyCheckTestRule.java

```diff
@@ -461,7 +461,8 @@
             @Node(coordinates = @Loc(value = AREA_LOCATION_3)),
             @Node(coordinates = @Loc(value = LOCATION_OUTSIDE_AREA_1)),
             @Node(coordinates = @Loc(value = AREA_LOCATION_BETWEEN_2_AND_3)) }, areas = {
-                    @Area(id = "100", coordinates = { @Loc(AREA_LOCATION_1), @Loc(AREA_LOCATION_2),
+                    @Area(id = "100", tags = "last_edit_time=20200422", coordinates = {
```

Comment:
This could be updated to now check for
- synthetic nearest neighbor tag
- missing iso tag